iml-manager-cli/Cargo.toml

```diff
@@ -19,6 +19,7 @@ structopt = "0.2"
 termion = "1"
 tokio = "0.1"
 exitcode = "1.1"
+reqwest = { version = "0.9", features = ["default-tls", "tls"] }
```

Comment:
Do you need this dep?

Reply:
I may not any more, i'll remove and re-compile.